extensions/amp-tiktok/0.1/test/validator-amp-tiktok.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
```

Comment:
this is missing the license agreement that is [at the top of files](https://github.com/ampproject/amphtml/blob/main/extensions/amp-3d-gltf/0.1/test/validator-amp-3d-gltf.html). Please be sure the year matches this year (`2021`) since this is being added now.